app/policies/organisation_policy.rb

```diff
@@ -4,6 +4,32 @@ def index?
   end
 
   def show?
-    true
+    user.administrator? || user.fund_manager? || associated_user?
+  end
+
+  def create?
+    user.administrator? || user.fund_manager?
+  end
+
+  def update?
+    user.administrator? || user.fund_manager? || associated_user?
```

Comment:
Users who are associated with an organisation should be able to update the information.